cypress/e2e/ui/menu.cy.js

```diff
@@ -5,64 +5,491 @@ describe('Menu', () => {
+        });
+
+        it('Configured Systems', () => {
+          cy.menu('Automation', 'Configuration', 'Configured Systems').expect_show_list_title('Configured Systems');
+        });
+      });
+
+      describe('Embedded Ansible', () => {
+        it('Playbooks', () => {
+          cy.menu('Automation', 'Embedded Ansible', 'Playbooks').expect_show_list_title('Playbooks (Embedded Ansible)');
+        });
+
+        it('Repositories', () => {
+          cy.menu('Automation', 'Embedded Ansible', 'Repositories').expect_show_list_title('Repositories (Embedded Ansible)');
+        });
+
+        it('Credentials', () => {
+          cy.menu('Automation', 'Embedded Ansible', 'Credentials').expect_show_list_title('Credentials');
+        });
+      });
+
+      describe('Embedded Automate', () => {
+        it('Explorer', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Explorer').expect_explorer_title('Datastore');
+        });
+
+        it('Simulation', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Simulation').expect_explorer_title('Simulation');
+        });
+
+        it('Generic Objects', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Generic Objects').expect_explorer_title('All Generic Object Definitions');
+        });
+
+        it('Customization', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Customization').expect_explorer_title('All Dialogs');
+        });
+
+        it('Import / Export ', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Import / Export').expect_show_list_title('Import / Export');
+        });
+
+        it('Log', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Log').expect_show_list_title('Log');
+        });
+
+        it('Log', () => {
+          cy.menu('Automation', 'Embedded Automate', 'Requests').expect_show_list_title('Requests');
+        });
+      });
+    });
+
+    describe('Control >', () => {
+      it('Policy Profiles', () => {
+        cy.menu('Control', 'Policy Profiles').expect_show_list_title('Policy Profiles');
+      });
+
+      it('Policies', () => {
+        cy.menu('Control', 'Policies').expect_show_list_title('Policies');
+      });
+
+      it('Events', () => {
+        cy.menu('Control', 'Events').expect_show_list_title('Events');
+      });
+
+      it('Conditions', () => {
+        cy.menu('Control', 'Conditions').expect_show_list_title('Conditions');
+      });
+
+      it('Actions', () => {
+        cy.menu('Control', 'Actions').expect_show_list_title('Actions');
+      });
+
+      it('Alert Profiles', () => {
+        cy.menu('Control', 'Alert Profiles').expect_show_list_title('Alert Profiles');
+      });
+
+      it('Alerts', () => {
+        cy.menu('Control', 'Alerts').expect_show_list_title('Alerts');
+      });
+
+      it('Simulation', () => {
+        // cy.menu('Control', 'Simulation').expect_show_list_title('');
+      });
```

Comment:
Can we mark this one as "pending" instead of commenting out?  Otherwise it appears 🟢 .
```suggestion
      it.skip('Simulation', () => {
        cy.menu('Control', 'Simulation').expect_show_list_title('');
      });
```